### PR TITLE
README.md is out of date

### PR DESCRIPTION
**Pull request by ress997**
`--quiet` flag has changed https://github.com/aquasecurity/trivy/pull/125

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1245,25 +1244,27 @@ Example: https://github.com/aquasecurity/trivy-ci-test/blob/master/Dockerfile
...
+  --quiet, -q                 suppress progress bar and log output
+  --no-progress               suppress progress bar
   --ignore-unfixed            display only fixed vulnerabilities
   --refresh                   refresh DB (usually used after version update of trivy)
   --auto-refresh              refresh DB automatically when updating version of trivy
   --debug, -d                 debug mode
-  --vuln-type value           comma-separated list of vulnerability types (os,library)
-  --cache-dir value           cache directory (default: "/path/to/cache")
+  --vuln-type value           comma-separated list of vulnerability types (os,library) (default: "os,library")
+  --cache-dir value           cache directory (default: "~/.cache/trivy")
```
Comment (knqyf263) [inline README.md]: This default value depends on the machine running Trivy. In my environment (macOS), `default: "/Users/knqyf263/Library/Caches/trivy"`.

File: README.md [hunk 2]
```diff
@@ -1094,8 +1093,8 @@ before_install:
   - wget https://github.com/aquasecurity/trivy/releases/download/v${VERSION}/trivy_${VERSION}_Linux-64bit.tar.gz
   - tar zxvf trivy_${VERSION}_Linux-64bit.tar.gz
 script:
-  - ./trivy --exit-code 0 --severity HIGH --quiet --auto-refresh trivy-ci-test:${COMMIT}
-  - ./trivy --exit-code 1 --severity CRITICAL --quiet --auto-refresh trivy-ci-test:${COMMIT}
+  - ./trivy --exit-code 0 --severity HIGH -no-progress --auto-refresh trivy-ci-test:${COMMIT}
```
Comment (knqyf263) [inline README.md]:
```suggestion
  - ./trivy --exit-code 0 --severity HIGH --no-progress --auto-refresh trivy-ci-test:${COMMIT}
```
I know that `-no-progress` works as well.

File: README.md (hunk 1 above)
Comment (ress997) [reply README.md]: OK

Return to original format

File: README.md (hunk 2 above)
Comment (ress997) [reply README.md]: Sorry, I misspelled it.

**Comment by ress997** [comment]
@knqyf263  I corrected it, so please check it.

**Comment by knqyf263** [comment]
Many thanks!

**Pull request merged**